README.md

```diff
@@ -445,11 +445,15 @@ matrix(
 Prettier ships with a handful of customizable format options, usable in both the CLI and API.
 
 ### Print Width
-Specify the length of line that the printer will wrap on.
-
-**We strongly recommend against using more than 80 columns.**
-
-Prettier works by cramming as much content as possible until it reaches the limit, which happens to work well for 80 columns but makes lines that are very crowded. When a bigger column count is used in styleguides, it usually means that code is allowed to go beyond 80 columns, but not to make every single line go there, like Prettier would do.
+Specify the line length that the printer will wrap on.
+
+> **For readability we recommend against using more than 80 characters:**
+>
+>In code styleguides, maximum line length rules are often set to 100 or 120.
+>
+>This is not the amount of characters that will always be on a line. For readability a developer will use whitespace to break up long lines so the average amount of characters will be 80 characters on a line.
+>
+>Prettier will put the maximum amount of characters on a line. With the print width set to 120 every line would have around 110 - 120 characters. This is harder to read than the recommended 80 characters.
```

Comment:
Thanks much.  IMHO, the change from `columns` to `characters` makes this less clear. 
I could not tell if the first sentence here was trying to indicate some fine detailed difference between _characters_/_columns_ per line and the `--print-width`.

>This is not the amount of characters that will always be on a line.

It looks like it is just saying that _people_ don't always put the max characters per line.  

Here's a suggested edit that, to me, clarifies the distinction between how humans and prettier handle max len rules:

>When humans write code, they don't strive to reach the maximum number of columns on every line. Developers often use whitespace to break up long lines for readability. In practice, the average line length often ends up well below the maximum.
>
>Prettier, on the other hand, strives to use the maximum number of columns per line. With the print width set to 120, every line would have around 110 - 120 columns. This may be harder to read than the recommended 80 columns.

Take it with a grain of salt as this is obviously just one man's grammatical opinion.